schemas/2015-10-01/Microsoft.Media.json

```diff
@@ -0,0 +1,84 @@
+{
+  "id": "http://schema.management.azure.com/schemas/2015-10-01/microsoft.media.json#",
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Microsoft.Media",
+  "description": "Microsoft Media Service Resource Type",
+  "resourceDefinitions": {
+    "mediaServices": {
+      "type": "object",
+      "properties": {
+        "type": {
+          "enum": [
+            "Microsoft.Media/mediaServices"
+          ]
+        },
+        "apiVersion": {
+          "enum": [
+            "2015-10-01"
+          ]
+        },
+        "name": {
+          "oneOf": [
+            {
+              "type": "string",
+              "pattern": "^[a-z0-9]{3,26}$"
+            },
+            {
+              "$ref": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression"
+            }
+          ]
+        },
+        "properties": {
+          "oneOf": [
+            {
+              "$ref": "#/definitions/storageAccounts"
+            },
+            {
+              "$ref": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression"
+            }
+          ]
+        }
+      },
+      "required": [
+        "type",
+        "apiVersion",
+        "location"
+      ],
+      "description": "Microsoft.Media/mediaServices"
+    }
+  },
+  "definitions" : {
+    "storageAccounts": {
+      "properties": {
+        "storageAccounts": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/storageAccount"
+          }
+        }
+      },
+      "type": "object"
+    },
+    "storageAccount": {
+      "properties": {
+        "id": {

```

Comment:
What’s the preferred way to push this change, update just the schema, or update all 4 files?

From: Brian Blum
Sent: Wednesday, June 22, 2016 11:56 AM
To: 'Azure/azure-resource-manager-schemas' <reply@reply.github.com>; Azure/azure-resource-manager-schemas <azure-resource-manager-schemas@noreply.github.com>
Cc: Author <author@noreply.github.com>
Subject: RE: [Azure/azure-resource-manager-schemas] Add Azure Media Service 2015-10-01 schema (#188)

Will add expression.

From: Wenjun Wu [mailto:notifications@github.com]
Sent: Wednesday, June 22, 2016 11:43 AM
To: Azure/azure-resource-manager-schemas <azure-resource-manager-schemas@noreply.github.com<mailto:azure-resource-manager-schemas@noreply.github.com>>
Cc: Brian Blum <brianbl@microsoft.com<mailto:brianbl@microsoft.com>>; Author <author@noreply.github.com<mailto:author@noreply.github.com>>
Subject: Re: [Azure/azure-resource-manager-schemas] Add Azure Media Service 2015-10-01 schema (#188)


In schemas/2015-10-01/Microsoft.Media.json<https://github.com/Azure/azure-resource-manager-schemas/pull/188#discussion_r68111151>:

> +  },

> +  "definitions" : {

> +    "storageAccounts": {

> +      "properties": {

> +        "storageAccounts": {

> +          "type": "array",

> +          "items": {

> +            "$ref": "#/definitions/storageAccount"

> +          }

> +        }

> +      },

> +      "type": "object"

> +    },

> +    "storageAccount": {

> +      "properties": {

> +        "id": {

please support expression for the "id" property, usually a string type property already support expression, but if there is pattern on it, we need to define $ref expression for it.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/Azure/azure-resource-manager-schemas/pull/188/files/93541a94f21f4a1b95abe1e38ca14477e4b57270#r68111151>, or mute the thread<https://github.com/notifications/unsubscribe/AEH9bHC0wUZ0j9U1s6VNBHez-hZ29FgFks5qOYIcgaJpZM4I8C67>.